docs/en/operations/monitoring.md

```diff
@@ -15,6 +15,27 @@ You can monitor:
 - Utilization of hardware resources.
 - ClickHouse server metrics.
 
+## Built-in observability dashboard
+
+<img width="400" alt="Screenshot 2023-11-12 at 6 08 58 PM" src="https://github.com/ClickHouse/ClickHouse/assets/3936029/2bd10011-4a47-4b94-b836-d44557c7fdc1">
+
+ClickHouse comes with a built-in observability dashboard feature which can be accessed by `$HOST:$PORT/dashboard` (requires user and password) that shows the following metrics:
```

Comment:
```suggestion
ClickHouse comes with a built-in observability dashboard feature which can be accessed by `$HOST:$PORT/dashboard` (requires username and password) that shows the following metrics:
```

Reply:
Should be _username_?

Reply:
It's a database `user`: https://clickhouse.com/docs/en/sql-reference/statements/create/user. The dashboard UI also references `user`